bmi_lstm.py

```diff
@@ -220,13 +232,46 @@ def update(self):
             
             self.scale_output()
             
-            self.t += 1
-    
+            self.t += self._time_step_size
```

Comment:
Given the time step is defined as 3600, this seems like it would produce a different outcome than `self.t += 1`.